project2/py/web_scraper.py

```diff
@@ -63,6 +65,19 @@ def extract(direction, collection):
             lst.append(word)
     return lst
 
+def get_primary_method(directions, methods):
+    # Go through all directions & return whichever primary cooking method is mentioned most frequently.
+    methods_dict = {}
+    for direction in directions:
+        direction = direction.lower().replace('.', '').split()
+        for word in methods:
+            if word in direction:
+                if word in methods_dict:
+                    methods_dict[word] += direction.count(word)
+                else:
+                    methods_dict[word] = 1
+    sorted_methods = sorted(methods_dict.items(), key=operator.itemgetter(1))
+    return [sorted_methods[0][0]]
```

Comment:
wait why is it a list?

Reply:
I was passing it as a string before, but the formatting done in `Recipe.py` made that string print out letter-by-letter, like:

Reply:
So I just put the string in a list to have the formatter print out the single-element list element-by-element

Reply:
if there's a better/more clever way to accomplish this, though, i'm all ears!